docs/decisions/0002-no-longer-rely-on-portal-designer.rst

```diff
@@ -0,0 +1,71 @@
+
+* Enterprise Customer uuid (from the ``EnterpriseCustomer`` model in ``edx-enterprise`` / LMS)
+* Enterprise Customer name
+* Learning coordinator email address (exposes to enterprise learners in the learner portal)
+* Branding configuration
+    * Logo image
+    * Banner colors (background/border)
+
+As mentioned above, ``designer`` is also used to configure the Programs learner portals for each Master's degree for a given university partner (i.e., site). However, the configuration data for Programs differs from the configuration data for an Enterprise. For example, the learner portal for a Program might be configured to show links to program-related documents, render static text in the sidebar of that program's learner portal, or use custom background images in the banner.
+
+Additionally, the ``designer`` service provides an API that is called during the build process for the Programs and Enterprise learner portals in order to gather the necessary configuration data for each Program or Enterprise learner portal.
+
+However, by configuring the Programs and Enterprise learner portals in the same ``designer`` service, we are introducing a bit of a tight coupling between Programs and Enterprise, despite the configuration data being different for each learner portal. Further, as time goes on, the configuration for Enterprise and Programs learner portals will likely continue to diverge.
+
+Data Duplication for Enterprise Customer Configuration
+======================================================
+
+By configuring the Enterprise learner portal data through ``designer``, we inevitably introduced data duplication between ``designer`` and the Enterprise Customer configuration that is done through Django admin in the LMS (via ``edx-enterprise``).
+
+The Enterprise Customer configuration done in Django admin already includes the majority of data the Enterprise learner portal relies on: the Enterprise Customer's uuid, name, and logo image. The same data needs to be configured in the ``designer`` service for each Enterprise Customer, leading to the data duplication across ``designer`` and the LMS.
+
+********
+Decision
+********
+
+Due to the challenges outlined above, the Enterprise learner portal will no longer rely on the ``designer`` service. Instead, the configuration data for an Enterprise learner portal will be added to the Django admin page within the LMS (via ``edx-enterprise``):
+
+* Branding configuration for the banner
+    * Fields for the background and border HEX color values will be added to the ``EnterpriseCustomerBrandingConfiguration`` model.
+* Learning coordinator email address
+    * A field for this data will be added to the ``EnterpriseCustomer`` model.
+* A field will be added to the ``EnterpriseCustomer`` model to specify which Enterprise Customers have the learner portal enabled.
```

Comment:
We've already done this so I would suggest this should be past tense, but I don't know what the standard is for ADRs written after the fact. Not a big deal either way

Reply:
Yeah, I wrote these ADRs with the framing of what the current state was at the time of making the decision and the impact of the decision afterwards as opposed to writing about how things have already changed. It was a bit easier to describe the direct impact of these decisions when talking in the future tense here. Thanks for looking out for consistent tense throughout the ADRs, though 😄